tests/playground.html

```diff
@@ -617,15 +618,7 @@ <h1>Blockly Playground</h1>
   Variables category uses untyped variable blocks.
   See https://developers.google.com/blockly/guides/create-custom-blocks/variables#untyped_variable_blocks for more information. -->
   <xml xmlns="https://developers.google.com/blockly/xml" id="toolbox-categories" style="display: none">
-    <category name="Logic" categorystyle="logic_category">
-      <block type="controls_if"></block>
-      <block type="logic_compare"></block>
-      <block type="logic_operation"></block>
-      <block type="logic_negate"></block>
-      <block type="logic_boolean"></block>
-      <block type="logic_null" disabled="true"></block>
-      <block type="logic_ternary"></block>
-    </category>
+    <category type="logic_category"></category>
```

Comment:
I worry, since this is the "main" playground toolbox, that folks would copy it without including the `playgrounds/toolbox_categories.js` file. Is there some other category we could use as an example?
If not, perhaps we could only use the registered logic category for toolbox-categories-typed-variables?

Reply:
I moved it to the typed-variables toolbox and added a comment above it. If you still think it is confusing lmk and I can try to come up with a better solution.